exportoptionsgenerator/targets.go

```diff
@@ -45,32 +93,14 @@ func ArchivableApplicationTarget(xcodeProj *xcodeproj.XcodeProj, scheme *xcschem
 	return &mainTarget, nil
 }
 
-func filterApplicationBundleTargets(targets []xcodeproj.Target, exportMethod exportoptions.Method) (filteredTargets []xcodeproj.Target) {
-	fmt.Printf("Filtering %v application bundle targets", len(targets))
-
+func filterApplicationBundleTargets(targets []xcodeproj.Target) (filteredTargets []xcodeproj.Target) {
 	for _, target := range targets {
 		if !target.IsExecutableProduct() {
 			continue
 		}
 
-		// App store exports contain App Clip too. App Clip provisioning profile has to be included in export options:
```

Comment:
Moved to exportoptionsgenerator.go, so the filtering of app clips is consistent for xcode-archive and export-xcarchive.